app/decorators/manageiq/providers/autosde/storage_manager/storage_manager_decorator.rb

```diff
@@ -0,0 +1,35 @@
+Module ManageIQ::Providers
+  class AutoSDE::StorageManagerDecorator < MiqDecorator
```

Comment:
Aside form AnsibleTower, this is the only provider specific entry in this repo - @skateman Is that accurate?  How does it work for other providers?  Should this live in the provider repo instead?

Reply:
Well, we have no decorators in provider repos ... yet. But I'm not against being this the first one.

Reply:
Resetteled this into the provider.